substratevm/src/com.oracle.svm.core.genscavenge/src/com/oracle/svm/core/genscavenge/graal/GenScavengeGCFeature.java

```diff
@@ -92,6 +94,7 @@ public void duringSetup(DuringSetupAccess access) {
 
         ManagementSupport managementSupport = ManagementSupport.getSingleton();
         managementSupport.addPlatformManagedObjectSingleton(java.lang.management.MemoryMXBean.class, new HeapImplMemoryMXBean());
+        managementSupport.addPlatformManagedObjectList(java.lang.management.MemoryPoolMXBean.class, Arrays.asList(new YoungGenerationMemoryPoolMXBean(), new OldGenerationMemoryPoolMXBean()));
```

Comment:
HotSpot uses three pools for the serial GC: `eden`, `survivor`, and `old`. I think we should do the same.

`GenScavengeGCFeature` is used for the serial and for the epsilon GC. For the epsilon GC, only a single memory pool should be added (like on HotSpot).